source/index.ts

```diff
@@ -344,6 +344,13 @@ export default class PQueue<QueueType extends Queue<EnqueueOptionsType> = Priori
 		return this._queue.size;
 	}
 
+	/**
+	Size of the queue filtered by options.
+	*/
```

Comment:
Can you also include the use-case? Users don't always realize how something could be useful to them, so it's nice to give them some hints.

Reply:
This also needs to be documented in the readme.